experimental/cache_size.cc

```diff
@@ -0,0 +1,65 @@
+#include <stdlib.h>
+#include <time.h>
+
+#include <algorithm>
+#include <iostream>
+
+int64_t MAX_SIZE = 16 * 1024 * 1024;
+int64_t MIN_SIZE = 1024;
+int64_t ACCESSES_NUM = 64 * 1024 * 1024;
+int REPEATS = 50;
+
+clock_t whack_cache(const int64_t sz, int64_t* accesses) {
```

Comment:
Let's add a comment about this function's goal and its approach. Alternatively, a comment at the top of the file about what's going on will help.

Reply:
Will add comments.